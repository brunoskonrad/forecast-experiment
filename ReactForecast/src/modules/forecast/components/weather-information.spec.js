import React from 'react'
import TestRenderer from 'react-test-renderer'

import WeatherInformation from './weather-information'

describe('Modules', () => {
  describe('Forecast', () => {
    describe('Components', () => {
      describe('Forecast Information', () => {
        it('should render component', () => {
          const weather = {
            city: 'Berlin',
            status: 'Clear sky',
            temperature: 28
          }
          const subject = TestRenderer.create(
            <WeatherInformation weather={weather} />
          )

          expect(subject.toJSON()).toMatchSnapshot()
        })
      })
    })
  })
})
