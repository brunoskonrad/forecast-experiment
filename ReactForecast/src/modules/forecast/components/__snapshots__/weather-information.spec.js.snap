// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modules Forecast Components Forecast Information should render component 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontSize": 34,
        },
        undefined,
      ]
    }
  >
    Berlin
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontSize": 20,
          "marginTop": 4,
        },
        undefined,
      ]
    }
  >
    Clear sky
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontSize": 60,
          "fontWeight": "200",
          "marginTop": 8,
        },
        undefined,
      ]
    }
  >
    28
    Â°
  </Text>
</View>
`;
